tests/cli/test_cli.py

```diff
@@ -180,25 +147,6 @@ def test_export_cli_no_data_ignore(tmpdir):
         )
 
 
-@pytest.mark.skip(reason="Needs refactoring from path to uri.")
-def test_export_cli_unsupported_region(tmpdir):
-    with pytest.raises(NotImplementedError):
-        _ = CliRunner().invoke(
-            hydromt_cli,
-            [
-                "export",
-                str(tmpdir),
-                "-s",
-                "hydro_lakes",
-                "-r",
-                "{'subbasin': [-7.24, 62.09], 'uparea': 50}",
```

Comment:
How would we export for a certain region? I `hydromt export` should still be tested right?

Reply:
The reason (as far as I understand) that the region got removed from the cli, is that the way we construct regions has become much more complicated and interwoven with the components. This means that regions can no longer be specified independent of what type of components you have. The `export` can be used without a region and then all the data will be exported. aside from that we also have a file format for specifying what data needs to get exported. I'm not sure if this is updated yet, but regions will have to be specified there. There should be at least one test that uses the file format.